### PR TITLE
Theme keylines

### LINKED ISSUE #556: Theme component - key lines
[ORANGE]

Key lines should separate article columns, not the top row and bottom row

screen shot 2017-02-17 at 10 36 52

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Fixes #556 

dividers

### CODE REVIEW

File: server/views/pages/explore.njk [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
   <div class="row row--small-top-padding">
     <div class="container">
-      <div class="grid">
+      <div class="grid grid--dividers">
```
Comment (davidpmccormick) [inline server/views/pages/explore.njk]: I think this class isn't set up to handle multiple rows of items very well – looks like the first divider is hidden by a `:first-child` selector, but the first on the second row has a divider to the left.

File: server/views/components/series-container/index.njk [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="container">
-    <div class="grid">
+    <div class="grid grid--dividers">
```
Comment (jakubfiala) [inline server/views/components/series-container/index.njk]: The `.grid--dividers` class seems like a pretty reusable thing, why not turn it into a mixin and use different classes to denote parts of the grid that need the dividers? Something like `grid--articles`, which could then have additional styling independent of the dividers.

Comment (jamesgorrie) [inline server/views/components/series-container/index.njk]: I prefer composing what my content will look like in the HTML by adding CSS classes.

Creating a coupling of css style => css component => html component leaves a lot more room to break things.

From my experience in the past, especially with large sites, being stupid with CSS is a good rule (that's basically what BEM is).

Comment (jakubfiala) [inline server/views/components/series-container/index.njk]: sounds good 👍

**Comment by jamesgorrie** [comment]
@jakubfiala due to [@davidpmccormick's comment](https://github.com/wellcometrust/wellcomecollection.org/pull/563#discussion_r101776282) - I cant merge just yet

**Comment by jakubfiala** [comment]
no problemo

File: server/views/pages/explore.njk (hunk 1 above)
Comment (davidpmccormick) [inline server/views/pages/explore.njk]: Could you split them into two rows of four?

Comment (jamesgorrie) [inline server/views/pages/explore.njk]: But they might be rows of 1/2/3/4 on mobile etc...

Comment (davidpmccormick) [inline server/views/pages/explore.njk]: Ah yeah, of course.

**Comment by davidpmccormick** [comment]
Suggest closing this in favour of #607.

**Comment by jamesgorrie** [comment]
@davidpmccormick Does this solve the keyline issue?

**Comment by davidpmccormick** [comment]
> Does this solve the keyline issue?

It does.

**Pull request closed without merging**